backoffice/src/Models/Domain/Droit.cs

```diff
@@ -1,7 +1,59 @@
-namespace Droits.Models
+namespace Droits.Models;
+
+public class Droit
 {
-    public class Droit
-    {
-       public Guid Id{get;set;}
-    }
-}
+    // Base fields...
+
+    public Guid Id { get; set; }
+
+    public string? Reference { get; set; } // This is the current reference.
+    public DroitStatus Status { get; set; } = DroitStatus.Received;
+    public DateTime ReportedDate { get; set; } = DateTime.UtcNow;
+    public DateTime Created { get; set; } = DateTime.UtcNow;
+    public DateTime Modified { get; set; } = DateTime.UtcNow;
+
+
+    // public List<DroitItem> WreckMaterials {get;set;} = new List<DroitItem>();
+
+    // Wreck
+
+    public Guid? WreckId { get; set; }
+
+    public bool IsHazardousFind { get; set; } = false;
+
+    // Wreck Vessel
+
+    public string? WreckVesselName { get; set; } = string.Empty;
+    public string? WreckConstructionDetails { get; set; } = string.Empty;
+    public int? WreckVesselYearConstructed { get; set; }
+    public int? WreckVesselYearSunk { get; set; }
+
+
+    // Location
+    public string? Latitude { get; set; }
+    public string? Longitude { get; set; }
+    public bool InUkWaters { get; set; } = false;
+    public int? LocationRadius { get; set; } //Units? Unknown.
+    public int? Depth { get; set; } //Units? Unknown. //metres from frontend
+    public string? LocationDescription { get; set; } = string.Empty;
+
+
+    // Salvage
+
+    public bool SalvageAwardClaimed { get; set; } = false;
+    public string? ServicesDescription { get; set; }
+    public int? ServicesDuration { get; set; } //Units? Unknown.
```

Comment:
Are we going to have domain models separate from data structures that we use e.g as API responses/entity models?

Reply:
Not sure the best approach at the moment, but it's worth thinking about now